packages/linked-data-proof/src/__tests__/Ed25519Signature2018.digitalbazaar/__fixtures__/dereferencingDocumentLoader.ts

```diff
@@ -0,0 +1,77 @@
+import { contexts } from "./contexts";
+import controller from "./controller.json";
+import jsonld from "jsonld";
+
+const contextResolver = async (iri: string) => {
+  if (contexts[iri]) {
+    return { document: contexts[iri] };
+  }
+  return undefined;
+};
+
+const documentResolver = async (iri: string) => {
+  if (iri.startsWith(controller.id)) {
+    return {
+      documentUrl: controller.id,
+      document: controller
+    };
+  }
+  return undefined;
+};
+
+// Note: you implement this without using JSON-LD frame,
+// but it MUST return a document with an @context.
+const documentDereferencer = async (document: any, iri: string) => {
+  try {
+    const frame = await jsonld.frame(
```

Comment:
hmm do we really want to use frame here? can we refactor jsonld out of this function?

Reply:
This has been removed.